tools/istio-iptables/main.go

```diff
@@ -0,0 +1,523 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/joho/godotenv"
+
+	dep "istio.io/istio/tools/istio-iptables/pkg/dependencies"
+)
+
+func getEnvWithDefault(key string, defaultValue string) string {
+	value, exists := os.LookupEnv(key)
```

Comment:
We have a small env variable library that does this and automatically generates documentation. For example https://github.com/istio/istio/blob/34f5ff08d13fdb7589459cc8b3090f974ec465f4/pkg/features/pilot/pilot.go#L45.

There is supposed to be a linter blocking os.LookupEnv but probably doesn't run in tools